CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 - Rename custom request header to `aptos-typescript-sdk`
 - [`Breaking`] Rename `token` to `digitalAsset` and add digital asset built in transaction generation functions
 - [`Breaking`] change transaction submission builder flow namespace to be under a `transaction` namespace
+- Rename `SingleSignerTransaction` type to `SimpleTransaction`
```

Comment:
I'd put a `[Breaking]` here too

Reply:
dont think it is breaking tho, the inner type hasnt changed but just the type name. non of the tests and/or examples have been affected by it

Reply:
Oh, just in the event someone is using the type name, since if you use the hardcoded type, rather than just inferring it is breaking.